cmd/validator/flags/flags.go

```diff
@@ -45,6 +45,12 @@ var (
 		Usage: "Beacon node REST API provider endpoint.",
 		Value: "http://127.0.0.1:3500",
 	}
+	// BeaconRESTApiHeaders defines a list of headers to send with all HTTP requests to the beacon node.
+	BeaconRESTApiHeaders = &cli.StringFlag{
+		Name: "beacon-rest-api-headers",
```

Comment:
The name is a little verbose but I wanted to keep it aligned with the `--beacon-rest-api-provider` flag